wordpress-mobile/WordPress-Android
# Gravatar Library - Remove Okio dependency from WPAndroid

-----

## To Test:



We've moved the Gravatar upload to an external library. We no longer need the Okio dependency in this project. Verify that you can still upload new avatars.

-----

## Regression Notes

1. Potential unintended areas of impact

    - Nothing. We've just moved some code to an external library and removed the unnecessary code.

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - Don't apply

3. What automated tests I added (or what prevented me from doing so)

    - Don't apply

-----
